vaadinfrontend/src/main/resources/messages.properties

```diff
@@ -1,4 +1,5 @@
-GENERAL=Ooooooops! Sorry! -> Please contact support@qbic.zendesk.com.
+GENERAL=Apologies, something went wrong -> Please contact support@qbic.zendesk.com.
 INVALID_PROJECT_TITLE=The project title {1} is invalid. -> A valid project title is not empty and is shorter than {0} characters.
 INVALID_EXPERIMENTAL_DESIGN=The experimental design description is invalid -> A vaild description of an experimental design must not be empty and contain at most {0} characters.
 INVALID_PROJECT_OBJECTIVE=The project objective {0} is invalid. -> A valid objective is not empty.
+MISSING_PROJECT_CONTEXT=The project context is unknown but was expected -> Please contact support@qbic.zendesk.com
```

Comment:
There error codes should only be something the user understands. I do not think that `missing project context` provides any information to the average user. Sounds more like a developer error message. IMHO this should go to the logs instead.